### PR TITLE
feat: add directorys with fonts, conect fonts

### PR DESCRIPTION
**Pull request by ruslan-milinchuk**
Connected  #5

### CODE REVIEW

File: src/components/ProfileSlider/index.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+          <a className="person__link" href={link}>
+            Enquire about {name[0]}
+          </a>
+        </div>
+      </div>
+    );
+  }
+
+  slideLeft = list => {
+    let i = this.state.i;
```
Comment (andriy-ilin) [inline src/components/ProfileSlider/index.js]: let {i} = this.state

File: src/components/ProfileSlider/index.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+class ProfileSlider extends Component {
+  state = { i: 0 };
+
+  render() {
+    const { list, name, role, link } = this.props;
+    return (
+      <div className="person">
+        <div className="person__content">
+          <div className="person__name">
+            <h2 className="person__name_font">{name[0]}</h2>
```
Comment (andriy-ilin) [inline src/components/ProfileSlider/index.js]: use <br/>

File: src/components/ProfileSlider/index.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+import "./style.css";
+import ArrowLeft from "../../icons/ArrowLeft";
+import ArrowRight from "../../icons/ArrowRight";
+import ButtonList from "../ButtonList";
+class ProfileSlider extends Component {
+  state = { i: 0 };
+
+  render() {
+    const { list, name, role, link } = this.props;
+    return (
```
Comment (andriy-ilin) [inline src/components/ProfileSlider/index.js]: const [firstName, lastName] = name

File: src/components/ProfileSlider/index.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
+import React, { Component } from "react";
+import "./style.css";
+import ArrowLeft from "../../icons/ArrowLeft";
+import ArrowRight from "../../icons/ArrowRight";
+import ButtonList from "../ButtonList";
+class ProfileSlider extends Component {
+  state = { i: 0 };
```
Comment (andriy-ilin) [inline src/components/ProfileSlider/index.js]: i => currentIndex

File: src/components/ProfileSlider/index.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+                className="person__btn-right"
+              >
+                <ArrowRight />
+              </button>
+            </div>
+          </div>
+        </div>
+        <div className="person__img">
+          <img
+            src={list[this.state.i]}
```
Comment (andriy-ilin) [inline src/components/ProfileSlider/index.js]: const {i} = this.state

File: src/components/ProfileSlider/index.js [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+            Enquire about {name[0]}
+          </a>
+        </div>
+      </div>
+    );
+  }
+
+  slideLeft = list => {
+    let i = this.state.i;
+    if (i === 0) {
```
Comment (andriy-ilin) [inline src/components/ProfileSlider/index.js]: use logic operators

return i>0 ? this.setState({ i: i - 1 }) : i === 0 ? this.setState({ i: list.length - 1 }) : 'something else'

File: src/components/ProfileSlider/index.js [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+      return this.setState({ i: list.length - 1 });
+    }
+    if (i > 0) {
+      return this.setState({ i: i - 1 });
+    }
+  };
+  stateNumb = index => {
```
Comment (andriy-ilin) [inline src/components/ProfileSlider/index.js]:
```
stateNumb = i=> {
  return this.setState({ i });
}
```

File: src/components/ProfileSlider/index.js [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+    return this.setState({ i: index });
+  };
+
+  slideRight = list => {
+    let i = this.state.i;
```
Comment (andriy-ilin) [inline src/components/ProfileSlider/index.js]: const {i} = this.state

File: src/components/ButtonList/index.js [hunk 9]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+const ButtonList = ({ list, stateNumb }) => {
+  return list.map((item, index) => (
+    <button
+      key={index}
+      onClick={() => stateNumb(index)}
+      className="person__btn-item"
+    >
+      0{index + 1}
```
Comment (andriy-ilin) [inline src/components/ButtonList/index.js]: check if `index +1 >= 10` don't write `zero` before number

File: src/components/ProfileSlider/index.js (hunk 7 above)
Comment (ruslan-milinchuk) [reply src/components/ProfileSlider/index.js]: It doesn't work, I don't know why

**Pull request merged**